docs/tutorials/stateless-application/hello-minikube.md

```diff
@@ -291,6 +291,52 @@ Run your app again to view the new message:
 minikube service hello-node
 ```
 
+## Enable Add-ons
+
+Minikube has a set of built in addons that can be used enabled, disabled, and opened inside of the local Kubernetes environment. Below is an example of this functionality for the `heapster` addon.
+
+First we can list the currently supported addons:
+
+```shell
+$ minikube addons list
+- storage-provisioner: enabled
+- kube-dns: enabled
+- registry: disabled
+- registry-creds: disabled
+- addon-manager: enabled
+- dashboard: disabled
+- default-storageclass: enabled
+- coredns: disabled
+- heapster: disabled
+- efk: disabled
+- ingress: disabled
+```
+
+Minikube must be running for these command to take effect addon:
+
+```shell
+$ minikube addons enable heapster
+heapster was successfully enabled
+
```

Comment:
The style in this document is to show a command without the $ prompt. Then have a sentence that says something about the command output. Then show the output.

Enable the heapster addon:

```shell
minikube addons enable heapster
```

The output verifies that heapster is enabled:

```
hepaster was successfully enabled
```